components/frontend/src/utils.js

```diff
@@ -193,28 +202,39 @@ sortWithLocaleCompare.propTypes = {
     strings: stringsPropType,
 }
 
+function hideMetric(metric, metricsToHide, hiddenTags) {
+    const hideBecauseNoActionNeeded =
+        metricsToHide === "no_action_needed" && ["target_met", "debt_target_met", "informative"].includes(metric.status)
+    const hideBecauseNoIssues = metricsToHide === "no_issues" && !metric.issue_ids
+    const hideBecauseTagIsHidden =
+        hiddenTags?.length > 0 &&
+        hiddenTags?.filter((hiddenTag) => metric.tags?.includes(hiddenTag)).length >= metric.tags?.length
+    return hideBecauseNoActionNeeded || hideBecauseNoIssues || hideBecauseTagIsHidden
+}
+visibleMetrics.propTypes = {
```

Comment:
This propType still needs to be changed :)

Reply:
Oops. Weird I didn't see any error.